IntAct.pm

```diff
@@ -0,0 +1,334 @@
+
+=head1 DESCRIPTION
+
+ A VEP plugin that retrieves molecular interaction data for variants as reprted by IntAct database.
+ 
+ Please cite the IntAct publication alongside the VEP if you use this resource:
+ https://pubmed.ncbi.nlm.nih.gov/24234451/
+ 
+ Pre-requisites:
+ 
+ 1) The IntAct data files can be downloaded from -
+ http://ftp.ebi.ac.uk/pub/databases/intact/current/various
+ 
+ 2) The genomic location mapped file needs to be tabix indexed. You can 
+ do this by following commands -
+
+  a) filter, sort and then zip
+  grep -v -e '^$' -e '^[#\-]' mutation_gc_map.txt | sed '1s/.*/#&/' | sort -k1,1 -k2,2n -k3,3n | bgzip > mutation_gc_map.txt.gz
+  
+  b) create tabix indexed file -
+  tabix -s 1 -b 2 -e 3 -f mutation_gc_map.txt.gz
+
+ 3) As you have already noticed, tabix utility must be installed in your path to use this plugin.
+ 
+ Options are passed to the plugin as key=value pairs:
+
+ file				: (mandatory) Path to tabix-indexed genomic coordinate file
+ mutation_file			: (mandatory) Path to IntAct data file
+ default			: (redundant) Set value to 1 to include feature_type, interaction_participants, 
+				  pmid, and interaction_ac in the output.
+ minimal		  	: Set value to 1 to overwrite default option to include only interaction_ac 
+				  in the output
+ 
+ you can also customize output using the following options -
+ feature_ac			: Set value to 1 to include Feature AC in the output
+ feature_short_label		: Set value to 1 to include Feature short label in the output
+ feature_range			: Set value to 1 to include Feature range(s) in the output
+ original_sequence		: Set value to 1 to include Original sequence in the output
+ resulting_sequence		: Set value to 1 to include Resulting sequence in the output
+ feature_type			: Set value to 1 to include Feature type in the output
+ feature_annotation		: Set value to 1 to include Feature annotation in the output
+ ap_ac				: Set value to 1 to include Affected protein AC in the output
+ ap_symbol			: Set value to 1 to include Affected protein symbol in the output
+ ap_full_name			: Set value to 1 to include Affected protein full name in the output
+ ap_organism			: Set value to 1 to include Affected protein organism in the output
+ interaction_participants	: Set value to 1 to include Interaction participants in the output
+ pmid				: Set value to 1 to include PubMedID in the output
+ figure_legend			: Set value to 1 to include Figure legend in the output
+ interaction_ac			: (redundant) Set value to 1 to include Interaction AC in the output. Adding
+ 				  this option redundant as it is already included by default or mimimal options.
+ 
+ See what this options mean - https://www.ebi.ac.uk/intact/download/datasets#mutations
+
+ Notice:
+ 
+ Currently this plugin is tested on human data only, other species data will be reported but should be used with caution. 
```

Comment:
What does this mean?

Reply:
Assuming this is related to the issue with $config->{species} mentioned above, then by filtering to only use human data by default this statement can be removed. Future work on this plugin could include allowing other species data to be "switched on" instead of human.

Reply:
This line is removed.